src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/GlobalNavigationHubLinks.java

```diff
@@ -30,7 +30,7 @@ public void TestHubLinksInGlobalNav_001_clickHubsLinks() {
             String link = globalNav.getHubLink(hub);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "link" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1481)